weather/www/css/style.css

```diff
@@ -0,0 +1,144 @@
+/*http://stackoverflow.com/questions/11777598/font-size-relative-to-the-users-screen-resolution
+http://stackoverflow.com/questions/24933858/how-to-customize-font-size-in-ionic-framework*/
+ion-content{
+  background: url("night.jpg");
+  background-repeat:no-repeat;
+  background-size:cover;
+
+}
+.col {
+  background-color: #707070 !important;
+  opacity: .6;
+  /* This sets the background color*/
+}
+.item {
+  background-color: #707070 !important;
+  opacity: .7;
+  /* This sets the background color*/
+}
+
+.mid {
+  text-align: center;
+}
+@media (min-height: 0) {
```

Comment:
I don't think you need to make this a media query. The properties you set inside this media query will always be executed because a device always has a hight of at least zero.